corgi/tasks/errata_tool.py

```diff
@@ -79,6 +95,10 @@ def slow_load_errata(erratum_name):
             # once all build's components are ingested we must save product taxonomy
             sb = SoftwareBuild.objects.get(build_id=b)
             sb.save_product_taxonomy()
+        # Note this makes a permanent record in Redis and needs to removed before this task can
+        # be processed again with the same erratum_name. Use force_process true option to clear it,
+        # Or delete the redis persistent storage
+        set_task_complete(f"load_errata:{erratum_name}")
```

Comment:
I don't love this design. Can't we bail on the fact that the erratum is already loaded in the database? Interacting directly with Redis just introduces another layer of communication between workers and Redis outside of the native Celery stuff.

Reply:
this was closed